untitled-sdl-tank-game/PatrollingEnemy.cpp

```diff
@@ -0,0 +1,79 @@
+#include "PatrollingEnemy.h"
+
+PatrollingEnemy::PatrollingEnemy(float x, float y, float collisionRadius, MyTexture * texture, Point destinationPoint, Point patrolPointA, Point patrolPointB, double minDinstanceForChase)
+	: Enemy(std::move(x), std::move(y), std::move(collisionRadius), std::move(texture)),
+	m_patrolPointA{ std::move(patrolPointA) },
+	m_patrolPointB{ std::move(patrolPointB) },
+	m_minDinstanceForChase{ std::move(minDinstanceForChase) },
```

Comment:
is `std::move()` really needed here ?, it makes the code look complicated :hear_no_evil: